### PR TITLE
feat: temporarily inject examples for mutually-exclusive request types in c8 api docs

### PR DESCRIPTION
**Pull request by pepopowitz**
## Description

Follow-up to https://github.com/camunda/camunda-docs/pull/4308#discussion_r1765350148.

* Temporarily injects example requests for two endpoints, each of which has requests that contain mutually exclusive fields. 
* Also removes a description hack we were using for an unrelated field, because it's no longer needed.

Following this PR, I'll open one at the camunda/camunda repo, to add these examples at the source. It won't be merged until after the release though, due to the current code freeze. Thus, this temporary hack, so that we don't have to keep modifying the spec when generating the docs for now. 

### Create process instance endpoint

image

image

### Evaluate decision endpoint

image

image

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

- [x] My changes require a [docs infrastructure review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process).

### CODE REVIEW

File: api/camunda/generation-strategy.js [hunk 1]
```diff
@@ -4,7 +4,18 @@ const outputDir = "docs/apis-tools/camunda-api-rest/specifications";
 const specFile = "api/camunda/camunda-openapi.yaml";
 
 function preGenerateDocs() {
-  hackChangesetDescription();
```
Comment (pepopowitz) [inline api/camunda/generation-strategy.js:7]: This description hack is no longer needed!

File: api/camunda/generation-strategy.js [hunk 2]
```diff
@@ -17,32 +28,45 @@ module.exports = {
...
+                  variables: {}
+              "By BPMN process ID":
+                summary: "Create a process instance by bpmnProcessID and version."
+                value:
+                  bpmnProcessId: "1234-5678"
+                  version: 1
+                  variables: {}
+      responses:`,
+  };
+}
```
Comment (tmetzke) [inline api/camunda/generation-strategy.js:51]: I'll backport the changes into the release branch. This shouldn't be necessary then, I guess?

**Review by tmetzke — APPROVED** [review]
Great examples, thanks a lot, @pepopowitz 🦕 
The hacks for the examples can be removed, I think. I'll add the source changes to the release branch as well.

File: api/camunda/generation-strategy.js (hunk 1 above)
Comment (tmetzke) [reply api/camunda/generation-strategy.js:7]: 🥳

File: api/camunda/generation-strategy.js (hunk 2 above)
Comment (tmetzke) [reply api/camunda/generation-strategy.js:51]: Done with https://github.com/camunda/camunda/pull/22715

**Review by christinaausley — APPROVED** [review]

**Comment by pepopowitz** [comment]
> Great examples, thanks a lot, @pepopowitz 🦕 The hacks for the examples can be removed, I think. I'll add the source changes to the release branch as well.
> 
> _Edit_: examples integrated in the source, also on release branch (with [camunda/camunda#22715](https://github.com/camunda/camunda/pull/22715))

Nice, thanks @tmetzke!!! I didn't expect that this would be backport-worthy, and I thought I'd have to wait until the freeze thawed. 

Closing!!! I'll address the removal of `hackChangesetDescription` in another upcoming PR. (It currently has no effect, so there's no harm in leaving it.)

**Pull request closed without merging**